### PR TITLE
Chainbridge wizard

### PR DESCRIPTION
**Pull request by GregTheGreek**
This PR adds a interactive shell for interacting with chainbridge.

### Testing locally
- You will need a valid json wallet (go to MeW :) ) and run a local ganache instance with that key having the funds
- run `$ node ./wizard/`

### Features needed to merge PR:
- [x] Collect chain information
- [x] Collect bridge deployment information (required parameters for deploying bridge contract)
- [x] Select handler contracts to deploy
- [x] Save configurations to home directory
- [x] Load previous configurations
- [x] Deploys selected contracts
- [x] Deletes a chain from the config
- [x] Updates configuration with deployed addresses
- [x] Accepts privatekey / json file for deployment
- [x] Choose to interact with a specific chain based on saved configuration
- [x] Allow selective deployment of contracts from your config

##### Substrate support
- This can be done by prompting the user at the beginning which type of chain they want to interact with (eg: ethereum or substrate)
##### Interact with the contracts
- By leveraging the saved configuration file we can easily interact with the deployed contracts
- Update previous configurations

Requires: #20

### CODE REVIEW

File: cb-sol-cli/wizard/questions/deploy.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                
+            ]
+        }
+    ],
+    bridge: [
+        {
+            type: "number",
+            name: "fee",
+            message: "What fee would you like to charge? (in ETH)",
+            validate: x => x <= 0 ? "Number must be greater than 0" : true
```
Comment (ansermino) [inline cb-sol-cli/wizard/questions/deploy.js:49]: Should allow it to be 0

Comment (ansermino) [reply cb-sol-cli/wizard/questions/deploy.js:49]: Forces integers so min is 1eth

File: cb-sol-cli/wizard/questions/deploy.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        relayerNumber: {
+                type: "number",
+                name: "numRelayers",
+                message: "How many relayers do you want? (number)",
+                validate: x => x <= 0 ? "Number must be greater than 0" : true
+            },
+        relayerThreshold: {
+                type: "number",
+                name: "relayerThreshold",
+                message: "What threshold do you want? (number)"
```
Comment (ansermino) [inline cb-sol-cli/wizard/questions/deploy.js:74]:
```suggestion
                message: "What vote threshold do you want? (number)"
```

File: cb-sol-cli/wizard/questions/deploy.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            name: "chainId",
+            message: "What is the ChainBridge chainId?",
+            validate: x => x <= 0 ? "Number must be greater than 0" : true
+        }
+    ],
+    relayer: {
+        relayerNumber: {
+                type: "number",
+                name: "numRelayers",
+                message: "How many relayers do you want? (number)",
```
Comment (ansermino) [inline cb-sol-cli/wizard/questions/deploy.js:68]: Saying "number" seems redundant

File: cb-sol-cli/wizard/questions/deploy.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                        // an invalid address will throw
+                        ethers.utils.getAddress(values[i]);
+                    } catch (e) {
+                        return `${values[i]} is not a valid Ethereum address!`
+                    }
+                }
+                return true;
+            }
+        }
+    }
```
Comment (ansermino) [inline cb-sol-cli/wizard/questions/deploy.js:94]:
```
? Enter relayer addresses (Comma separated values) ›  0xff93B45308FD417dF303D6515aB04D9e89a750Ca,0x8e0a907331554AF72563Bd8D43051C2E64Be5d35,0x24962717f8fA5BA3b931bACaF9ac03924EB475a0
›  0xff93B45308FD417dF303D6515aB04D9e89a750Ca is not a valid Ethereum address!

```

**Review by ansermino — CHANGES_REQUESTED** [review]
Can we show defaults for options?

File: cb-sol-cli/wizard/questions/deploy.js (hunk 4 above)
Comment (GregTheGreek) [reply cb-sol-cli/wizard/questions/deploy.js:94]: Cannot reproduce

File: cb-sol-cli/wizard/questions/deploy.js (hunk 1 above)
Comment (ansermino) [reply cb-sol-cli/wizard/questions/deploy.js]: Still forces integer

File: cb-sol-cli/wizard/questions/deploy.js (hunk 4 above)
Comment (ansermino) [reply cb-sol-cli/wizard/questions/deploy.js:94]:
```
? Enter relayer addresses (Comma separated values) › "0xf78b3E93d744e233594A797A00f87451cd236E53,0x89A183Cd4679Ba3195D3f93a105c643c9075aC97,0x4a0c39336b2e0976EA50Da6C992E672e87546e7B"
› "0xf78b3E93d744e233594A797A00f87451cd236E53 is not a valid Ethereum address!
```

**Pull request closed without merging**